### PR TITLE
(feat): command+enter/ctrl+enter to make the request

### PR DESCRIPTION
**Pull request by IgorHalfeld**
I just implement #109 :tada:

### CODE REVIEW

**Comment by IgorHalfeld** [comment]
ping @fenos

**Comment by IgorHalfeld** [comment]
Hey @fenos, this feature will really improve my life hahaha

**Comment by fenos** [comment]
Lovely PR, Thanks @IgorHalfeld

**Comment by fenos** [comment]
@IgorHalfeld Why the package-lock.json changed?

File: app/components/Editor/Editor.tsx [hunk 1]
```diff
@@ -143,6 +145,16 @@ export function Editor({ protoInfo, initialRequest, onRequestChange }: EditorPro
     metadata: (initialRequest && initialRequest.metadata) || getMetadata() || INITIAL_STATE.metadata,
   }, undefined);
 
+  const commands: Command[] = [
+    {
+      name: 'Request',
+      bindKey: { win: 'Ctrl+Enter', mac: 'Command+Enter' },
+      exec: () => {
+        makeRequest({ protoInfo, state, dispatch });
```
Comment (jackielii) [inline app/components/Editor/Editor.tsx:153]: This won't work as the state will be a reference to the old state. Tested, and fixes in https://github.com/uw-labs/bloomrpc/pull/215

**Pull request merged**